index.html

```diff
@@ -1389,12 +1399,12 @@ <h3>Issuance Date</h3>
         <dl>
           <dt><var>issuanceDate</var></dt>
           <dd>
-A <a>verifiable credential</a> MUST have an <code>issuanceDate</code> <a>property</a>.
-The value of the <code>issuanceDate</code> <a>property</a> MUST be a string
-value of an [[!RFC3339]] combined date and time string representing the date
-and time the <a>credential</a> was issued. Note that this date represents the
-earliest date when the information associated with the
-<code>credentialSubject</code> <a>property</a> became valid.
+A <a>credential</a> MUST have an <code>issuanceDate</code> <a>property</a>. The
```

Comment:
This is a non-substantive change as the group meant this property to apply to both "credentials" and "verifiable credentials".

Reply:
@grantnoble I'm just noting here why this change is ok. You typically don't want to change normative statements (anything w/ MUST, SHOULD, MAY) during the Candidate Recommendation phase.

Reply:
Thanks!